src/provider/zeebe/properties/BusinessRuleImplementationProps.js

```diff
@@ -76,6 +76,9 @@ function BusinessRuleImplementation(props) {
 
     let extensionElement, extensionElementType;
 
+    resetElement(element, commandStack);
```

Comment:
The undo/redo chain is broken if we have this as additional command.

If you for example:
1. Switch to DMN
2. Then switch to Job Worker
3. Then Undo
4. => will result in "none", where it should result in "DMN"

Can you please:
* Execute this within 1 command
* add test case for this bug